### ISSUE TITLE
Support directories

### ISSUE THREAD

**Issue by oswaldoacauan** [csscomb/grunt-csscomb#14]
Today to comb a whole directory(1-1) we need something like this

```
files: [{
  expand: true,
  cwd: '/assets/css',
  src: ['**/*.css'],
  dest: '/assets/css',
  filter: 'isFile'
}]
```

Maybe we can add a support to diretories, since comb.js have `processDirectory` to do most of the work for us

```
files: [
  '/assets/css'
]
```

**Comment by paazmaya** [comment]
:+1:

**Comment by lhtdesignde** [comment]
+1 is this possible already?

**Comment by wardpeet** [comment]
@lhtdesignde this should fix the issue https://github.com/csscomb/grunt-csscomb/pull/42

**Comment by noahcooper** [comment]
Fixed in v4.0.0.
